pkg/kube/snapshot/snapshot_alpha.go

```diff
@@ -97,11 +107,28 @@ func (sna *SnapshotAlpha) Get(ctx context.Context, name, namespace string) (*v1a
 
 // Delete will delete the VolumeSnapshot and returns any error as a result.
 func (sna *SnapshotAlpha) Delete(ctx context.Context, name, namespace string) error {
+	snap, err := sna.Get(ctx, name, namespace)
```

Comment:
`Delete` now also gets and deletes the underlying `VolumeSnapshotContent`.

`NotFound` errors are ignored